education/cryptocurrency/risk.md

```diff
@@ -14,10 +14,10 @@ Education of how cryptocurrency technology works, how one coin is unique from ot
+While cryptocurrencies vary in supply (limited or unlimited); price stability is based on supply and demand on trading exchanges as well as a person's ability to buy or sell a lot of a specific coin. Since cryptocurrencies differ in design, investors should consider things like coin supply caps and amount of coins the founders are holding (or releasing at a later date) for project advancement purposes. Just like fiat currencies, prices are also impacted by factors outside of the cryptocurrency's control such as major news and announcements, or the coin's dependence on another cryptocurrency.
 
 ### Coin Security
-How an individual stores and secures his or her cryptocurrencies also determines investment risk. Those who are knowledgable about blockchain technology may increase coin security by ensuring their coins are saved locally on their computer using password-protected wallets. Passive investors however may leave their coins on an exchange wallet and treat the coin like  stock sitting in an E-Trade account. Although exchanges are becoming more reliable and user-friendly, investors should understand that exchanges facilitate an enormous amount of fiat currency and cryptocurrency on a dialy basis. Therefore, exchanges are prone to higher security threats like other money transmitters and alternative storage methods such as local hard drive wallets can be pursued. It should be noted that exchanges vary in their insurance policies for digital and fiat currencies. The Coinbase exchange was among the first to receive FDIC insurance for U.S. residents' fiat currency, protecting up to a maximum of $250,000 per investor.
+How a person stores and secures their cryptocurrencies also determines their investment risk. People who are knowledgable about blockchain technology may increase coin security by saving their coins on their own computer using password-protected wallets. Other investors might leave their coins on an exchange wallet and treat the coin like stock sitting in an E-Trade account. Although exchanges are becoming more reliable and user-friendly, investors should understand that exchanges trade huge amounts of fiat currency and cryptocurrency on a dialy basis. Because of this, exchanges are prone to higher security threats (it's kind of like 100 years ago when banks were threatened by theives). It is important to understand that exchanges vary in their insurance policies for digital and fiat currencies. The exchange Coinbase was among the first to receive FDIC insurance for U.S. residents' fiat currency, protecting up to a maximum of $250,000 per investor.
```

Comment:
did either of you find confusion among synonyms for cryptocurrency? (coin, crypto-coin, digital currency, etc.) Something I struggled with in writing various parts of cryptocurrency sections was trying to use synonyms instead of sounding repetitive. Just want to make sure same terms are used across all sections so people aren't misguided